calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Identify vulnerabilities in Kubernetes clusters.
+---
+
+# Automatically Scan images for vulnerabilities using Cluster Scanner
+
+## Big picture
+
+Automatically scan kubernetes workloads for vulnerabilities using {{prodname}} Image Assurance Cluster Scanner.
+
+## Value
+
+{{prodname}} Image Assurances cluster scanner helps you scan workloads running inside the cluster for vulnerabilities without much configuration. With Cluster Scanner you can:
+
+- Scan Kubernetes workload images for vulnerabilities automatically.
+- View vulnerabilities for the workloads in the {{prodname}} management UI
+- Assess the impact of newly-found vulnerabilities and prioritize remediation efforts
+- Create exceptions to ignore specific vulnerabilities
+- Create alerts on high-severity vulnerabilities so you can delegate remediation efforts to the appropriate team
+
+## About Image Assurance
+
+Image Assurance is based on the Common Vulnerabilities and Exposures (CVE) system, which provides a catalog of publicly-known security vulnerabilities and exposures. Known vulnerabilities are identified by a unique CVE ID based on the year it was reported (for example, CVE-2021-44228).
+
+Image Assurance provides a cluster scanner (deployed as a daemon set) to scan images. When enabled inside the cluster, it automatically scans images for vulnerabilities and displays them in management UI. Scanned image content includes:
+
+- Libraries and content (for example, python, ruby gems, jars and go)
+- Packages (OS and non-OS)
+- Image layer
```

Comment:
@penkeysuresh I wonder if we could perhaps standardize this section between the three pages and then create a separate mdx template that we can import into each of the three pages. Single source of truth.